plugin/trino-mysql/src/main/java/io/trino/plugin/mysql/MySqlClient.java

```diff
@@ -221,6 +237,99 @@ public PreparedStatement getPreparedStatement(Connection connection, String sql)
         return statement;
     }
 
+    @Override
+    public List<JdbcColumnHandle> getColumns(ConnectorSession session, JdbcTableHandle tableHandle)
```

Comment:
Let's add override of this method (without any modificiations, just a copy), as a separate commit, so it's possible to later identify what has been  changed since then